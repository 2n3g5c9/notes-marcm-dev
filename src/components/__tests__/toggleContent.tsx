import React from 'react'
import renderer from 'react-test-renderer'

import { ToggleContent } from '@components'

describe('ToggleContent', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ToggleContent />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
