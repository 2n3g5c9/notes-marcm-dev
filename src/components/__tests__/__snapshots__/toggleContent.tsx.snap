// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleContent renders correctly 1`] = `
Array [
  <button
    className="text-left text-sm sm:text-base"
    onClick={[Function]}
  >
    ▶️
      
    Show placeholder
  </button>,
  <div
    className="hidden"
  />,
]
`;
